Add locale en_IE / English (Ireland)

2ndkauboy opened the pull request:
## What?
Add the missing locale en_IE for English spoken in Ireland.

2ndkauboy committed:
Add locale en_IE / English (Ireland)

amieiro commented on locales/locales.php:
```diff
@@ -750,6 +750,17 @@ public function __construct() {
+		$en_ie->lang_code_iso_639_2 = 'eng';
+		$en_ie->lang_code_iso_639_3 = 'eng';
+		$en_ie->country_code = 'ie';
+		$en_ie->wp_locale = 'en_IE';
```
```suggestion
```
This should only be set for locales that are officially supported on translate.wordpress.org.

amieiro committed:
Remove the wp_locale

This should only be set for locales that are officially supported on translate.wordpress.org.

amieiro committed:
Merge branch 'develop' into locale-en-ie
